src/app/client/src/app/modules/home/component/main-home/main-home.component.ts

```diff
@@ -5,6 +5,7 @@ import { CoursesService, UserService, PlayerService } from '@sunbird/core';
 import { ResourceService, ToasterService, ServerResponse, ConfigService, UtilService} from '@sunbird/shared';
 import {  IInteractEventObject, IInteractEventEdata, IImpressionEventInput } from '@sunbird/telemetry';
 import * as _ from 'lodash';
+import * as $ from 'jquery';
```

Comment:
please justify the use of Jquery in angular code

Reply:
removed